contracts/aevm.md

```diff
@@ -23,19 +23,22 @@ Interaction with the first class objects of the Æternity chain (e.g.
 oracles, names, and state channels) is done through calls to a
```

Comment:
The necessity to call the built-in contract at address zero in order to interact with the primops makes unnecessary a statement that "the primop would check that ... value =< contract-balance.